app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  
+  before_filter :require_logged_in
+
+  def index
+    @relationships = current_user.following_relationships
+  end
+
+  def destroy
+    relationship = Relationship.find(params[:id])
+    relationship.delete if relationship.follower == current_user
```

Comment:
Just so you know, `delete` bypasses any callbacks your model could have. If you want the callbacks to be executed, then you need to use `destroy`.